shell/synchronize/etc/datf-share.cfg

```diff
@@ -1,6 +1,6 @@
-source_dir="/groups/umcg-solve-rd/scr01/datf/${data_group}/share"
+source_dir="groups/umcg-solve-rd/datf/${data_group}/share"
 destination_dir="/groups/solve-rd/prm10/sftp/datf/${data_group}/"
-destination_machine='umcg-${ATEAMBOTUSER}@cher-ami.hpc.rug.nl'
+destination_machine='nb-dt'
```

Comment:
The machine was also renamed in the inventory nb-dt -> nb-transfer to match the DNS entry.